File: prak_lat1 java/Disk.java
public class Disk{

	// Attributes
	private String type;
	private int capacity;
	private int price;

	// Constructor
	public Disk(String type, int capacity, int price){
		this.type = type;
		this.capacity = capacity;
		this.price = price;
	}

	// Set & Get methods
	public void setType(String type){
		this.type = type;
	}

	public void setCapacity(int capacity){
		this.capacity = capacity;
	}

	public void setPrice(int price){
		this.price =price;
	}

	public String getType(){
		return this.type;
	}

	public int getCapacity(){
		return this.capacity;
	}

	public int getPrice(){
		return this.price;
	}

}

File: prak_lat1 cpp/Disk.cpp
#include <iostream>
#include <string.h>

using namespace std;

class Disk{
	//Attributes
	private:
		string type;
		int capacity;
		int price;

	public:
		//Constructor
		Disk(){}
		Disk(string type, int capacity, int price){
			this->type = type;
			this->capacity = capacity;
			this->price = price;
		}

		// Set and Get methods

		void setType(string type){
			this->type = type;
		}

		void setCapacity(int capacity){
			this->capacity = capacity;
		}

		void setPrice(int price){
			this->price = price;
		}

		string getType(){
			return this->type;
		}

		int getCapacity(){
			return this->capacity;
		}

		int getPrice(){
			return this->price;
		}

		~Disk(){}
};